mono/mini/mini-wasm-debugger.c

```diff
@@ -427,6 +428,17 @@ mono_wasm_set_breakpoint (const char *assembly_name, int method_token, int il_of
 	return req->id;
 }
 
+EMSCRIPTEN_KEEPALIVE int
```

Comment:
gboolean?

Reply:
Since this is called by JS, which won't get a boolean, being explicit about values makes sense?